stretch_core/nodes/stretch_driver

```diff
@@ -1,42 +1,29 @@
 #! /usr/bin/env python
 
-from __future__ import print_function, division
```

Comment:
Same comment as before. Let's keep this on all files to help with a future Python 3 transition.